Formula/n/netpbm.rb

```diff
@@ -24,6 +24,7 @@ class Netpbm < Formula
     sha256 x86_64_linux:  "389525c1edd732598087f85a27752cf394815d6984321285f77941a0086ba366"
   end
 
+  depends_on "python@3.13" => :build
```

Comment:
Can we do `uses_from_macos "python"` here?

Reply:
Oh, good point, I'll try that.